security/auth/maskinporten.md

```diff
@@ -0,0 +1,141 @@
+---
+description: >
+  Enabling `server-2-server` authentication with external agencies using Maskinporten.
+---
+
+# Maskinporten Client
+
+{% hint style="warning" %}
+**Status: Opt-In Open Beta**
+
+This feature is only available in the [Google Cloud Platform (GCP)](../../clusters/gcp.md) clusters.
+{% endhint %}
+
+## Abstract
+
+Maskinporten is a service that offers a simple API security model based on the OAuth2 protocol, and the use of JWT bearer grants. A Concept inspired by [Google's System Accounts](https://developers.google.com/identity/protocols/oauth2/service-account).
+
+Maskinporten allows API providers (external agencies) to define access to their APIs, modeled as scopes and based on the consumer's organization number.
```

Comment:
Maybe swap the order of these paragraphs. Link to digdirs documentation/page? Should clearly describe the intended purposes of the service in the context of NAV (why/when do I need this?)